reVX/wind_setbacks/wind_setbacks.py

```diff
@@ -0,0 +1,1386 @@
+# -*- coding: utf-8 -*-
+"""
+Handler to convert exclusion to/from .h5 and .geotiff
+"""
+from abc import ABC, abstractstaticmethod, abstractclassmethod
+from concurrent.futures import as_completed
+from earthpy import clip as cl
+import fiona
+import geopandas as gpd
+import h5py
+import json
+import logging
+import numpy as np
+import os
+from rasterio import features
+import re
+from shapely.geometry import shape
+from warnings import warn
+
+from rex.utilities import parse_table, SpawnProcessPool
+from reV.handlers.exclusions import ExclusionLayers
+
+logger = logging.getLogger(__name__)
+
+
+class BaseWindSetbacks(ABC):
+    """
+    Create exclusions layers for wind setbacks
+    """
+    MULTIPLIERS = {'high': 3, 'moderate': 1.1}
+
+    def __init__(self, excl_h5, hub_height, rotor_diameter, regs_fpath=None,
+                 multiplier=None, hsds=False, chunks=(128, 128)):
+        """
+        Parameters
+        ----------
+        excl_h5 : str
+            Path to .h5 file containing or to contain exclusion layers
+        hub_height : float | int
+            Turbine hub height (m), used along with rotor diameter to compute
+            blade tip height which is used to determine setback distance
+        rotor_diameter : float | int
+            Turbine rotor diameter (m), used along with hub height to compute
+            blade tip height which is used to determine setback distance
+        regs_fpath : str | None, optional
+            Path to wind regulations .csv file, if None create global
+            setbacks, by default None
+        multiplier : int | float | str | None, optional
+            setback multiplier to use if wind regulations are not supplied,
+            if str, must one of {'high': 3, 'moderate': 1.1},
```

Comment:
if str must be one of those keys? Kind of unclear. Maybe say that `if str must be a key in cls.MULTIPLIERS`?

Also what is the multiplier used for? I think based on our conversations and my fuzzy memory that this multiplies the max blade tip height to determine the setback, but that would be worthwhile to include in this description.